src/modules/tests/test-data/cat-home-test/vehicle-checks/vehicle-checks.cat-home-test.reducer.ts

```diff
@@ -0,0 +1,69 @@
+
+import { CatFUniqueTypes } from '@dvsa/mes-test-schema/categories/F';
+import { CatGUniqueTypes } from '@dvsa/mes-test-schema/categories/G';
+import { CatHUniqueTypes } from '@dvsa/mes-test-schema/categories/H';
+import { CatKUniqueTypes } from '@dvsa/mes-test-schema/categories/K';
+
+import * as vehicleChecksCatHomeTestActionTypes from './vehicle-checks.cat-home-test.action';
+import {
+  NUMBER_OF_TELL_ME_QUESTIONS as numberOfTellMeQuestions,
+}
+  from '../../../../../shared/constants/tell-me-questions/tell-me-questions.cat-be.constants';
+import {
+  NUMBER_OF_SHOW_ME_QUESTIONS as numberOfShowMeQuestions,
+}
+  from '../../../../../shared/constants/show-me-questions/show-me-questions.cat-be.constants';
+
+export type VechicleChecksUnion =
+  | CatFUniqueTypes.VehicleChecks
+  | CatGUniqueTypes.VehicleChecks
+  | CatHUniqueTypes.VehicleChecks
+  | CatKUniqueTypes.VehicleChecks;
+
+export const initialState: CatFUniqueTypes.VehicleChecks = {
+  tellMeQuestions: Array(numberOfTellMeQuestions).fill({}),
+  showMeQuestions: Array(numberOfShowMeQuestions).fill({}),
+};
+// TODO add all home type categories
+export function vehicleChecksCatHomeTestReducer(
+  state: CatFUniqueTypes.VehicleChecks = initialState,
+  action: vehicleChecksCatHomeTestActionTypes.Types): VechicleChecksUnion {
+  switch (action.type) {
+    case vehicleChecksCatHomeTestActionTypes.SHOW_ME_QUESTION_SELECTED:
```

Comment:
InitializeVehicleChecks should be here, this must be added in for the sprint to become unblocked.

Reply:
no longer applicable ^